app/src/androidTest/java/com/sdp/movemeet/view/profile/EditProfileActivityImagePickerTest.java

```diff
@@ -87,7 +87,7 @@ public void uploadImageToFirebaseStorage() {
         ImageView profileImage = activityRule.getActivity().findViewById(R.id.image_view_edit_profile_image);
         Image image = new Image(imageUri, profileImage);
         image.setDocumentPath(userImagePath);
-        ImageHandler.uploadImage(image, progressBar);
+        ImageHandler.uploadImage(image,activityRule.getActivity());
```

Comment:
Ok ok I see that you changed the signature of the method `uploadImage` of the `ImageHandler` class, I will analyse it down below 👍.